src/Tribe/Main.php

```diff
@@ -590,6 +592,9 @@ public function hooks() {
 
 		// Cart handling.
 		add_action( 'init', tribe_callback( 'tickets.commerce.cart', 'hook' ) );
+
+		// Theme Compatibility.
+		add_filter( 'body_class', tribe_callback( 'tickets.theme-compatibility', 'filter_body_class' ) );
```

Comment:
I think this might need to replace the usage of `Tribe__Tickets__Attendee_Registration__Template::theme_body_class()` filtering on body_class